app/views/gsi-iframe.html

```diff
@@ -0,0 +1,18 @@
+<body>
+  <!--
+      Publications should load this next script from:
+      https://news.google.com/swg/js/v1/swg-gaa.js
+    -->
+  <script src="<% swgGaaJsUrl %>"></script>
+
+  <script>
+    GaaGoogleSignInButton.show({
+      // Allow specific origins to get results from your Google Sign-In button.
+      allowedOrigins: ['http://localhost:8000', 'http://localhost:8080'],
+
+      // Specify your Google Sign-In Client ID.
```

Comment:
nit: Please add a comment as to how to get this ID (https://developers.google.com/identity/one-tap/web/guides/get-google-api-clientid or https://developers.google.com/identity/sign-in/web/sign-in#create_authorization_credentials)

Reply:
Added!